recipes-openxt/qemu-dm/qemu-dm-2.6.2/ahci-disable-ncq.patch

```diff
@@ -0,0 +1,13 @@
+Index: qemu-2.6.2/hw/ide/ahci.c
+===================================================================
+--- qemu-2.6.2.orig/hw/ide/ahci.c
++++ qemu-2.6.2/hw/ide/ahci.c
+@@ -488,7 +488,7 @@ static void ahci_reg_init(AHCIState *s)
+     s->control_regs.cap = (s->ports - 1) |
+                           (AHCI_NUM_COMMAND_SLOTS << 8) |
+                           (AHCI_SUPPORTED_SPEED_GEN1 << AHCI_SUPPORTED_SPEED) |
+-                          HOST_CAP_NCQ | HOST_CAP_AHCI;
++                          /*HOST_CAP_NCQ |*/ HOST_CAP_AHCI;
```

Comment:
Just delete and if you feel its needed, add a comment that it was removed and why